src/pages/diversity.tsx

```diff
@@ -0,0 +1,121 @@
+import { ContentCard, ContentLeft, ContentRight } from '../components/ContentBox'
+import ContentSection from '../components/ContentSection'
+import { Headline } from '../components/Headline'
+import { BodyContainer } from '../components/HomePage/BodyContainer'
+import { Image } from '../components/Image'
+import { Standfirst } from '../components/Standfirst'
+import { events } from '../components/DIEvents/events'
+import { DISecondaryHeadline } from '../components/DiversityInclusion/SecondaryHeadline'
+import { InnerContentBox, StyledContentBox, PersonIcon } from '../components/DiversityInclusion/ContentBox'
+import { ContentHeadline } from '../components/ContentHeadline'
+import { BodyText } from '../components/BodyText'
+import { SecondaryHeadline } from '../components/SecondaryHeadline'
+import { ContentBoxText, ContentBoxTitle, InnerContainer, StyledTextContainer } from '../components/DiversityInclusion/TextContainer'
+
+const DiversityPage = () => (
+  <IndexLayout>
+    <Page>
+      <BodyContainer>
+        <ContentSection>
+          <ContentLeft>
+            <Headline>Diversity & Inclusion</Headline>
+            <Standfirst>
+              We value and respect all differences in all people (seen and unseen) at the Guardian. Here are some of the ways in which we
+              help to promote a diverse and inclusive culture.
+            </Standfirst>
+          </ContentLeft>
+
+          <ContentRight className="contentRight">
+            <Image className="diImage" filename="images/home-about-gnm.jpg" />
+          </ContentRight>
+        </ContentSection>
+
+        <ContentSection borderBottom={false}>
+          <SecondaryHeadline>Recruitment</SecondaryHeadline>
+          <StyledContentBox>
+            <InnerContentBox>
+              <PersonIcon />
+              <ContentHeadline>Job advertisement and description screening</ContentHeadline>
+              <BodyText>
+                We use an augmented writing tool which uses a combination of natural language processing and data mining to screen our job
+                postings for unconscious bias, reveal gendered language and help our job ads to appeal to a more diverse range of
+                candidates.
+              </BodyText>
+            </InnerContentBox>
+            <InnerContentBox>
+              <PersonIcon />
+              <ContentHeadline>Blind CV reviews</ContentHeadline>
+              <BodyText>
+                We redact candidate's names and educational institutions from CVs before they are reviewed by members of the Engineering
+                team to avoid unconscious bias in the selection process.
+              </BodyText>
+            </InnerContentBox>
+            <InnerContentBox>
+              <PersonIcon />
+              <ContentHeadline>CV screening</ContentHeadline>
+              <BodyText>
+                Three members of the Engineering team independently review redacted CVs before meeting to discuss their decisions together.
```

Comment:
This has recently been reviewed down to two. Maybe for another PR, though?

```suggestion
                Two members of the Engineering team independently review redacted CVs before meeting to discuss their decisions together.
```